js/mobile_menu.js

```diff
@@ -201,6 +201,31 @@ function closed() {
   popWin.style.visibility = 'hidden';
 }
 
+function validateEmail(event) {
+  const email = document.forms[0].elements.Email.value;
+  const res = email.match(/[A-Z]/g);
+  const btn = document.querySelector('form #form_btn');
+  const container = document.querySelector('form #form-btn-container');
+  const child = document.createElement('h2');
+  let msg = '';
+
+  if (res) {
+    msg = 'Error: Please use only lower case for the email address';
+    event.preventDefault();
+  } else return;
```

Comment:
Requirement screenshot
![image](https://github.com/erickma1/Module1_Portfolio_Setup/assets/78251927/d433a981-58e7-4af6-8599-674bc3531ea5)

According to the project requirements, when we create the error message, we should stick to the design guidelines (e.g., font, colors, margins) of [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 🤗

#### Example Design, (I'm referring to the `error` styling, not the error message) 👍
![image](https://github.com/erickma1/Module1_Portfolio_Setup/assets/78251927/0afdf756-4645-46e8-b94b-2d93f7781747)

Your design
![image](https://github.com/erickma1/Module1_Portfolio_Setup/assets/78251927/9701dbd5-2aa4-4f9c-bac5-eee6c55ef080)

Please kindly adjust your design. 💪